### ISSUE TITLE
Prefixes with URDF and SRDF?

### ISSUE THREAD

**Issue by atyshka** [ros-industrial/universal_robot#378]
I have a UR10 mounted on a mobile base and I would like to create a common URDF for the robot. The issue I am having is both the UR10 and my base have a base_link specified in the URDF. I would like to prefix the UR10 URDF it with something like ur10_ to avoid the tf collisions, but then then I believe that would break the entire MoveIt config. Is there a way to add prefixes for these files so I don't have to hard-code the prefixes for MoveIt? I only ask because prefixes were built into the UR10 URDF but seem unusable.

**Comment by gavanderhoorn** [comment]
You refer to "urdf" multiple times, but we only provide a set of xacro (macro) files in this repository.

As to the MoveIt configuration: MoveIt configurations are setup-specific, and the ones we provide in this repository are examples / default ones that let users run quick tests.

Robots are almost never used in isolation and any changes to the kinematic chain and/or the environment will require a re-run of the MSA.

In summary: yes, you can add a prefix, that is what the `prefix` xacro macro argument is for. Yes, this will 'break' the MoveIt configuration pkgs, but that is not a problem, as you're not supposed to reuse them anyway.

> I only ask because prefixes were built into the UR10 URDF but seem unusable.

I don't understand this sentence.
